core/src/defaults/parser.rs

```diff
@@ -2178,34 +2185,39 @@ fn declaration_section(parser: &LLP) -> bool {
         (Some(TT::Op(OK::Equal(_)) | TT::Keyword(KK::Packed)), Some(TT::Keyword(KK::Class))) => {
             false
         }
+        (
+            _,
+            Some(TT::Keyword(
+                KK::Label
+                | KK::Const(_)
+                | KK::Type
+                | KK::Var(_)
+                | KK::Exports
+                | KK::ThreadVar
+                | KK::Begin
+                | KK::Asm
+                | KK::Class
+                | KK::Property
+                | KK::Function
+                | KK::Procedure
+                | KK::Constructor
+                | KK::Destructor
+                | KK::End
+                | KK::Implementation
+                | KK::Initialization
+                | KK::Finalization,
+            )),
+        ) => true,
         (
             _,
             Some(
                 TT::Keyword(
-                    KK::Label
-                    | KK::Const(_)
-                    | KK::Type
-                    | KK::Var(_)
-                    | KK::Exports
-                    | KK::ThreadVar
-                    | KK::Begin
-                    | KK::Asm
-                    | KK::Strict
+                    KK::Strict
                     | KK::Private
                     | KK::Protected
                     | KK::Public
                     | KK::Published
-                    | KK::Automated
-                    | KK::Class
-                    | KK::Property
-                    | KK::Function
-                    | KK::Procedure
-                    | KK::Constructor
-                    | KK::Destructor
-                    | KK::End
-                    | KK::Implementation
-                    | KK::Initialization
-                    | KK::Finalization,
+                    | KK::Automated,
```

Comment:
I love it when my comment get eaten by GitHub.

Would it make more sense and make this a bit cleaner to do something more like:
```suggestion
                (_, Some(TT::Keyword(kk) | TT::IdentifierOrKeyword(kk))) if matches!(kk, ...) => ... ,
```

Reply:
The render is confused too it would seem.

Reply:
Yeah good idea.